pepper-apis/studybuilder-cli/studies/lms/patches/germline-consent-addendum-email-events.conf

```diff
@@ -0,0 +1,137 @@
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "GERMLINE_CONSENT_ADDENDUM",
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "SENDGRID_EMAIL",
+        "templates": [
+          {
+            "emailTemplate": ${emails.germlineCompleted},
+            "language": "en",
+            "isDynamic": false
+          }
+        ],
+        "pdfAttachments": [{ "pdfName": "lmsproject-germline-consent-addendum", "generateIfMissing": true }]
+      },
+      "dispatchToHousekeeping": true,
+      "preconditionExpr": """user.studies["cmi-lms"].forms["GERMLINE_CONSENT_ADDENDUM"].questions["ADDENDUM_CONSENT_BOOL"].answers.hasTrue()""",
+      "order": 1
+    },
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "GERMLINE_CONSENT_ADDENDUM",
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "SENDGRID_EMAIL",
+        "templates": [
+          {
+            "emailTemplate": ${emails.germlineCompleted},
+            "language": "en",
+            "isDynamic": false
+          }
+        ],
+        "pdfAttachments": [{ "pdfName": "lmsproject-germline-consent-addendum-disagree", "generateIfMissing": true }]
+      },
+      "dispatchToHousekeeping": true,
+      "preconditionExpr": """user.studies["cmi-lms"].forms["GERMLINE_CONSENT_ADDENDUM"].questions["ADDENDUM_CONSENT_BOOL"].answers.hasFalse()""",
+      "order": 2
+    },
+
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "GERMLINE_CONSENT_ADDENDUM_PEDIATRIC",
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "SENDGRID_EMAIL",
+        "templates": [
+          {
+            "emailTemplate": ${emails.germlineCompletedParental},
+            "language": "en",
+            "isDynamic": false
+          }
+        ],
+        "pdfAttachments": [{ "pdfName": "lmsproject-germline-consent-addendum-pediatric", "generateIfMissing": true }]
+      },
+      "dispatchToHousekeeping": true,
+      "preconditionExpr": """user.studies["cmi-lms"].forms["GERMLINE_CONSENT_ADDENDUM_PEDIATRIC"].questions["ADDENDUM_CONSENT_BOOL_PEDIATRIC"].answers.hasTrue()""",
+      "order": 2
+    },
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "GERMLINE_CONSENT_ADDENDUM_PEDIATRIC",
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "SENDGRID_EMAIL",
+        "templates": [
+          {
+            "emailTemplate": ${emails.germlineCompletedParental},
+            "language": "en",
+            "isDynamic": false
+          }
+        ],
+        "pdfAttachments": [{ "pdfName": "lmsproject-germline-consent-addendum-pediatric-disagree", "generateIfMissing": true }]
+      },
+      "dispatchToHousekeeping": true,
+      "preconditionExpr": """user.studies["cmi-lms"].forms["GERMLINE_CONSENT_ADDENDUM_PEDIATRIC"].questions["ADDENDUM_CONSENT_BOOL_PEDIATRIC"].answers.hasFalse()""",
```

Comment:
How does unanswered evaluate in this context?

Reply:
Could you leverage something like: """user.studies["cmi-lms"].forms["GERMLINE_CONSENT_ADDENDUM_PEDIATRIC"].isStatus("COMPLETE")""" with the questions["ADDENDUM_CONSENT_BOOL_PEDIATRIC"].answers.hasFalse()""" to know that the user finished the activity and definitively answered "false"?

Reply:
you mean question NOT answered ?

Reply:
Here, Event will never trigger because its activity submission / completed event and the question is mandatory one.